### PR TITLE
[4.3.0] Add ICP userstore doc

### PR DESCRIPTION
**Pull request by DinithiDiaz**
## Purpose
$subject

This will fix https://github.com/wso2/docs-mi/issues/825.

### CODE REVIEW

**Comment by DinithiDiaz** [comment]
A screen capture of the rendered page is attached below:
[setting-up-a-userstore-in-icp.pdf](https://github.com/user-attachments/files/17142846/localhost-8000-en-latest-install-and-setup-setup-user-stores-setting-up-a-userstore-in-icp.pdf)

File: en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+       ```
+
+If you are already using a JDBC user store (database) with another WSO2 product ([WSO2 API Manager](https://wso2.com/api-management/), [WSO2 Identity Server](https://wso2.com/identity-and-access-management/), or an instance of [WSO2 Enterprise Integrator 6.x.x](https://wso2docs.atlassian.net/wiki/spaces/EI660/overview)), you can connect the same database to the ICP. Alternatively, you can create a new RDBMS user store and connect it to the ICP.
+
+1. To set up a new RDBMS, select the preferred RDBMS type and follow the instructions.
+
+	!!! Tip
+		If you already have an RDBMS user store set up, you can skip this step.
+
+	- [Setting up a MySQL database]({{base_path}}/install-and-setup/setup/databases/setting-up-mysql)
```
Comment (arunans23) [inline en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md:166]: Should we unify these pages to mention both MI an ICP?

Comment (DinithiDiaz) [reply en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md:166]: Updated the page structure via https://github.com/wso2/docs-mi/pull/962/commits/46fb5a3a15382eccf1551c3250b7c4653feb6f54.
updated-page-structure

**Review by arunans23 — APPROVED** [review]

File: en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
+# Configure a User Store in WSO2 Integration Control Plane
+
+## File-based user store (default)
+
+The default user store of the WSO2 Integration Control Plane (ICP) is file-based. You can open the `<ICP_HOME>/conf/deployment.toml` file and add new users to the file-based user store as shown below. You can [encrypt the plain text]({{base_path}}/install-and-setup/setup/security/encrypting-plain-text) using secure vault.
```
Comment (anupama-pathirage) [inline en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md:5]: Should have a comma before `as shown below`?  @DinithiDiaz

File: en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+		<td>
+			The URL for connecting to the LDAP. If you are connecting over LDAPS (secured LDAP), you need to import the certificate of the user store to the truststore (<code>client-truststore.jks</code> by default). See the instructions on how to <a href="{{base_path}}/install-and-setup/setup/security/importing-ssl-certificate/#importing-ssl-certificates-to-a-truststore">add certificates to the truststore</a>.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>connection_name</code>
+		</td>
+		<td>
+			The username used to connect to the user store and perform various operations. This user needs to be an administrator in the user store. That is, the user requires write permission to manage add, modify users and perform search operations on the user store. The value you specify is used as the DN (Distinguish Name) attribute of the user who has sufficient permissions to perform operations on users and roles in LDAP.
```
Comment (anupama-pathirage) [inline en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md:116]: Incorrect wording? @DinithiDiaz 

`The username used to` => `The username is used to`

`That is, the user requires write permission to manage add, modify users and perform search operations on the user store. ` => `That means the user requires write permission to manage, add, modify users, and perform search operations on the user store.`

File: en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+				<td>
+					The URL for connecting to the database. The type of database is determined by the URL string.
+				</td>
+			</tr>
+			<tr>
+				<td>
+					<code>username</code>
+				</td>
+				<td>
+					The username used to connect to the user store and perform various operations. This user needs to be an administrator in the user store. That is, the user requires write permission to manage add users, modify users, and perform search operations on the user store.
```
Comment (anupama-pathirage) [inline en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md:279]: Same as https://github.com/wso2/docs-mi/pull/962/files#r1797182403

**Pull request merged**